packages/stable/src/__tests__/api/groups.int.spec.ts

```diff
@@ -63,20 +46,6 @@ describe('Groups integration test', () => {
     });
   });
 
-  test('list service account', async () => {
```

Comment:
should these be re-added?

Reply:
No, both `apiKey` and `serviceAccount` apis are getting removed, we are not planning now to support these use cases